lms/envs/common.py

```diff
@@ -195,7 +195,7 @@
     # Enable URL that shows information about the status of variuous services
     'ENABLE_SERVICE_STATUS': False,
 
-    # Toggle to indicate use of a custom theme
+    # Toggle to indicate use of the Stanford theming system
```

Comment:
The reference to Stanford here seems related to @antoviaque and @tusbar's [thread below](https://github.com/edx/edx-platform/pull/8271/files#r37427045)